packages/ui-app/src/util/max.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright 2017-2019 @polkadot/ui-app authors & contributors
```

Comment:
@kwingram25 Added these to `@polkadot/util` here https://github.com/polkadot-js/common/pull/401 - however it is not pulled into apps master anyway, just something to keep in mind when/if we start using his derive functions as they are available